### PR TITLE
CB-12-stock-info

### PR DESCRIPTION
**Pull request by DHua5922**
* **Please check if the PR fulfills these requirements**
- [X] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)

New Feature: display stock information

* **What is the current behavior?What does the code do before you made the new changes?** 

No component for showing stock information

* **What is the new behavior?**

Component for showing stock information

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)

Used [date-fns](https://date-fns.org/). Users will need to [install date-fns](https://date-fns.org/docs/Getting-Started) with `npm install date-fns --save`. 

* **Other information**:

Once the API in the back end for getting stock information has been completed, this will code need to be updated by having it get stock information from the back end API.

### CODE REVIEW

**Comment by Codubee** [comment]
Hi Dylan, can you change the title of the PR to include the ticket number. Thank you!

**Comment by DHua5922** [comment]
> Hi Dylan, can you change the title of the PR to include the ticket number. Thank you!

PR title now includes the ticket number.

File: frontend/src/global.js [hunk 1]
```diff
@@ -18,6 +18,23 @@ export const navlinks = [
...
+        "Open Price": 261.07,
+        "t": 1582641000 
+    },
+    {
+        "c": 261.74,
+        "High Price": 263.31,
+        "Low Price": 260.68,
+        "Open Price": 261.07,
+        "t": 1582641000 
+    },
```
Comment (maahasakhia) [inline frontend/src/global.js:35]: just wondering why do you have this object twice? I rendered it, and it looks nice! The card with the stock info showed up twice, so I'm wondering if that was intentional

Comment (DHua5922) [reply frontend/src/global.js:35]: It's intentional. Right now, I am assuming the stock info will be a list of quotes. That is why I have the object twice.

**Review by sunnyguan — APPROVED** [review]
Looks good to me, good job on the tests!

**Review by Codubee — APPROVED** [review]
LGTM!

**Comment by Codubee** [comment]
Only thing left to do is fix the conflict and we can merge!

**Pull request merged**